session-replacement/src/main/java/com/amadeus/session/SessionManager.java

```diff
@@ -616,4 +619,10 @@ public String toString() {
     return new StringBuilder().append("SessionManager [namespace=").append(configuration.getNamespace()).append("]")
         .toString();
   }
+
+  public void remove(SessionData sessionData) {
+	logger.info("deleting session with sessionId: '{}'", sessionData.getId() );
```

Comment:
As we repeat 3 times 
```java
logger.info("deleting session with sessionId: '{}'", some-id);
deletedSessions.mark()
```
can you extract this into new markSessionDeletion method.

Reply:
fixed